packages/trace-viewer/src/ui/networkTab.tsx

```diff
@@ -26,171 +26,283 @@ import { GridView } from '@web/components/gridView';
 
 const renderCell = (entry: RenderedEntry, column: ColumnName) => {
   if (column === 'name')
     return <span title={entry.name.url}>{entry.name.name}</span>;
-  if (column === 'method')
-    return <span>{entry.method}</span>;
-  if (column === 'status')
-    return <span className={entry.status.className} title={entry.status.text}>{entry.status.code > 0 ? entry.status.code : ''}</span>;
-  if (column === 'contentType')
-    return <span>{entry.contentType}</span>;
-  if (column === 'duration')
-    return <span>{msToString(entry.duration)}</span>;
-  if (column === 'size')
-    return <span>{bytesToString(entry.size)}</span>;
-  if (column === 'start')
-    return <span>{msToString(entry.start)}</span>;
-  if (column === 'route')
-    return entry.route && <span className={`status-route ${entry.route}`}>{entry.route}</span>;
+  if (column === 'method') return <span>{entry.method}</span>;
+  if (column === 'status') {
+    return (
+      <span className={entry.status.className} title={entry.status.text}>
+        {entry.status.code > 0 ? entry.status.code : ''}
+      </span>
+    );
+  }
+  if (column === 'contentType') return <span>{entry.contentType}</span>;
+  if (column === 'duration') return <span>{msToString(entry.duration)}</span>;
+  if (column === 'size') return <span>{bytesToString(entry.size)}</span>;
+  if (column === 'start') return <span>{msToString(entry.start)}</span>;
+  if (column === 'route') {
+    return (
+      entry.route && (
+        <span className={`status-route ${entry.route}`}>{entry.route}</span>
+      )
+    );
+  }
 };
 
-const renderEntry = (resource: Entry, boundaries: Boundaries): RenderedEntry => {
+const renderEntry = (
+  resource: Entry,
+  boundaries: Boundaries
+): RenderedEntry => {
   const routeStatus = formatRouteStatus(resource);
   let resourceName: string;
   try {
     const url = new URL(resource.request.url);
     resourceName = url.pathname.substring(url.pathname.lastIndexOf('/') + 1);
-    if (!resourceName)
-      resourceName = url.host;
+    if (!resourceName) resourceName = url.host;
   } catch {
     resourceName = resource.request.url;
   }
   let contentType = resource.response.content.mimeType;
   const charset = contentType.match(/^(.*);\s*charset=.*$/);
-  if (charset)
-    contentType = charset[1];
+  if (charset) contentType = charset[1];
 
   return {
     name: { name: resourceName, url: resource.request.url },
     method: resource.request.method,
-    status: { code: resource.response.status, text: resource.response.statusText, className: statusClassName(resource.response.status) },
+    status: {
+      code: resource.response.status,
+      text: resource.response.statusText,
+      className: statusClassName(resource.response.status),
+    },
     contentType: contentType,
     duration: resource.time,
-    size: resource.response._transferSize! > 0 ? resource.response._transferSize! : resource.response.bodySize,
+    size:
+      resource.response._transferSize! > 0
```

Comment:
What about the following?

`resource.response._transferSize! || resource.response.bodySize`